lib/openshift/imagepruners_imageregistry_operator_openshift_io.rb

```diff
@@ -0,0 +1,7 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class ImageprunersImageregistryOperatorOpenshiftIo < ClusterResource
```

Comment:
Do we have to use such a long class name?

Reply:
I just copy from other resources, is it fine to define shorter?

Reply:
In order to be consistent with other resources, I suggest to keep it like this.

Reply:
@wzheng1 that's a lot of typing... I think it's fine for the `RESOURCE=...` line but for the class name, I think `ImagePruners` is sufficient.  WDYT @akostadinov ?

Reply:
I would even call it `ImagePrunersImageregistryOperatorOpenshiftIo`, then the short name will be `image_prunner` which reads much better.

Reply:
Using the full name for the class avoids conflicts like between `config_imageregistry_operator_openshift_io.rb` and `config_samples_operator_openshift_io.rb`. If we think that `ImagePrunners` will not conflict with anything, then class name can be shorter. Still keeping current `RESOURCE`.